python/tvm/relay/prelude.py

```diff
@@ -73,7 +73,7 @@ def get_tensor_array_shape(expr, dtype, prelude):
     return None
 
 
-def _get_name_static(canonical, dtype, shape):
+def _get_name_static(canonical, dtype, shape, batch_dim=None):
     """Get name for static shape tensor array op corresponding
     to the canonical name"""
```

Comment:
Better to improve the docstring to explain parameters, especially we now have `batch_dim`.